src/ggrc/utils/log_event.py

```
@@ -111,6 +111,9 @@ def log_event(session, obj=None, current_user_id=None, flush=True,
   if not revisions:
     return event
   if event is None:
+    if current_user_id is None:
+      from ggrc.models import all_models
+      current_user_id = all_models.Person.query.first().id
```

This is not correct because we don't want to set random users for certain events.

Are we even sure that current_user_id here would ever be None? if it is we could throw an exception instead of silently hiding a big bug.